src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,105 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = ",";
+    private static final int FIELDS = 2;
+    private String[] key = new String[FIELDS];
+    private Integer[] value = new Integer[FIELDS];
+    private int pos = 0;
+    
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            readFromCsv(fromFileName);
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage() + "Problem with input from CSV file");
+        } catch (NumberFormatException e) {
+            throw new RuntimeException(e.getMessage() + "Numeric parse exception");
+        }
+        try {
+            writeToCsv(toFileName);
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage() + "Problem with output to CSV file");
+        }
+    }
+
+    private void readFromCsv(String fromFileName) throws IOException, NumberFormatException {
+        BufferedReader bri = new BufferedReader(new FileReader(fromFileName));
```

Comment:
Usage `BufferedReader` and `FileReader` without try-catch-final or try-with-resources block may lead to unclosed streams. Refactor code.

Reply:
i did refactored